components/comments/comment-container.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticleId } from "../../utils/api";
+
+import SingleComment from "./comment-card";
+
+const Comments = () => {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id).then((receivedComments) => {
+      setComments(receivedComments);
+    });
+  }, []);
+
```

Comment:
You should consider adding user feedback in here that lets the user know when the comments are loading. When I switched to slow 3G in the browser the article loaded before the comments were ready.